### PR TITLE
Update PRINCIPLES.md

### PR DESCRIPTION
**Pull request by amye**
This updates PRINCIPLES with Leadership Principles from TOC.
A minimum of two weeks public comment period opens today, May 29th with a TOC vote to follow if we feel we've resolved any feedback.

### CODE REVIEW

**Comment by caniszczyk** [comment]
Thanks, love this.

If we also update the other principles we should get the GB to sign off also after the TOC approves (just like we did when these principles were first created)

**Comment by TheFoxAtWork** [comment]
the other principles will be updated before NA - they just happen to take a bit longer :)

**Comment by monadic** [comment]
LGTM

**Comment by dims** [comment]
LGTM

**Review by TheFoxAtWork — APPROVED** [review]
LGTM

File: PRINCIPLES.md
Comment (TheFoxAtWork) [inline PRINCIPLES.md:184]:
```suggestion
## Provide feedback and appreciation appropriately
```

File: PRINCIPLES.md
Comment (TheFoxAtWork) [inline PRINCIPLES.md:194]:
```suggestion
## Lead inclusively and empower all voices and opinions to be heard
```

**Review by raravena80 — APPROVED** [review]
👍

File: PRINCIPLES.md
Comment (jonjohnsonjr) [inline PRINCIPLES.md:191]:
```suggestion
#### Always do better
```

**Comment by amye** [comment]
> Some minor markdown issues. I _believe_ the heading hierarchy should look like this, but I may have misinterpreted the intent.

Accepted changes! It looked different when we were drafting it. 😁

File: PRINCIPLES.md
Comment (leonardpahlke) [inline PRINCIPLES.md:179]: > If a leader is not acting in accordance with these principles, community members should first address it with the other leaders of the project, TAG, committee, group, or body. Project and TAG governance documentation must explain procedures (including who is responsible) for response and resolution if these principles are not being followed. In general, if a leader is not following these principles, there should be a good faith effort to educate, inform, and provide the leader with an opportunity to improve before taking further action such as suspending or removing them from their leadership role. If the behavior that violates these principles persists, or if the other leaders of the applicable project, committee, or group do not take corrective action in response to a reported violation of these principles, projects should escalate the matter to the TOC.

I like this, but does the heading "Principles" best fit the above content? The heading "Principles" makes sense because it introduces the principles and defines them in subheadings. For the above content, perhaps it would be worth a separate heading? "Leadership accountability" or something similar? I can imagine someone looking for this information, and a separate heading highlights this better.

**Review by leonardpahlke — COMMENTED** [review]
I like it. One comment about a headline.

Is this a document that we refer to similar to the CoC in our governance? Or asked differently is this an extension to the CoC?

File: PRINCIPLES.md
Comment (craigbox) [inline PRINCIPLES.md:23]: nit: as this is a table of contents, could you indent these lines or make them a bulleted list in Markdown?  This would also be useful to do to the section above.

File: PRINCIPLES.md
Comment (craigbox) [inline PRINCIPLES.md:179]: > Project and TAG governance documentation must explain procedures (including who is responsible) for response and resolution if these principles are not being followed.

I understand these changes, and this section specifically, which uses the word "must", as imposing a new _requirement_ on all CNCF projects. 

If that is to be the case, I don't think that having these leadership principles documented in the "TOC Principles", which otherwise describes how the TOC operates, is "high-enough-level". My instinct is that it needs to be its own top-level document, similar to the CNCF Code of Conduct.

Could this be construed as imposing _governance_ on a project, something that the CNCF otherwise does not do?  (See the [Public Statement](https://github.com/cncf/toc/blob/main/PRINCIPLES.md#public-statement) in this document).

**Review by alessiodionisi — APPROVED** [review]
LGTM

File: PRINCIPLES.md
Comment (TheFoxAtWork) [inline PRINCIPLES.md:205]:
```suggestion
Handle disagreements respectfully. Seek to resolve conflict and disagreement in a respectful and collaborative manner and encourage others to do the same. Show empathy, care, and concern for others’ feelings, perspectives, and experiences while working towards a resolution that considers the Community, the projects we interact with, the mission of our work, and what can reasonably be accomplished with consensus in accordance with the project's governance and scope. For instance, a portion of a community presents a plan to completely refactor a project that dictates a stop work on all features. This could introduce contention and conflict within the community as many adopters and contributors are relying on those features. A reasonable resolution that may receive consensus from all members may be to refactor areas of the project where no feature development is occurring, complete outstanding features, and then refactor the remaining areas. Conflict and disagreement will happen, as a leader we must de-escalate, be the agent of calm, and provide neutrality for all parties to feel welcome in expressing their perspectives while pursuing an acceptable outcome. 
```

**Comment by amye** [comment]
We've made some changes to this, we'll be restarting public comment on this.
Please review, vote is scheduled to open on Github on September 20th.

File: PRINCIPLES.md
Comment (chris-short) [inline PRINCIPLES.md:175]: I like this. But, CNCF community is a little ambiguous and could be broadly or narrowly defined depending on the reader's knowledge. If the intent is to have this extend across the entire foundation's projects and subgroups or the TOC only, we probably need to be a little more precise with the language as I am not clear on the scope of CNCF community here.

It could be a one word update like "entire" if that's the scope.

File: PRINCIPLES.md
Comment (catblade) [inline PRINCIPLES.md:188]: Request to add something around "Make sure credit is given correctly, either in github references, or authorship locations".  Something around that?

File: PRINCIPLES.md
Comment (catblade) [inline PRINCIPLES.md:20]: Maybe under "Be welcoming" a section could be added for "train the next leaders?"  Identifying individuals who are helpful to the project/community consistently and helping them take over roles.  Give opportunities for others to help, if people are willing, and give them support.  Help them identify what roles do suit them and follow interests, and make sure not to ask for help just because someone is good at something but also because that is their interest.  Sometimes people get slotted into roles they have no interest in, like organizing or event planning, when really they are there for technical content, and we should be sensitive to that.

**Comment by tpepper** [comment]
One point I notice is missing is a notion of community over company.

**Comment by dims** [comment]
We can iterate again as needed!

**Comment by linsun** [comment]
LGTM, thanks @amye !

**Review by kevin-wangzefeng — COMMENTED** [review]
LGTM, thanks @amye

File: PRINCIPLES.md
Comment (craigbox) [reply PRINCIPLES.md:23]: https://github.com/cncf/toc/pull/1265

**Comment by craigbox** [comment]
> We can iterate again as needed!

Thanks @dims. There were a few unaddressed comments.  Could someone please make sure that these are carried forward?

I'm also not sure if you need to update the refreshed and/or sign-off date at the top of the document.  There may be an implication that the whole document was reviewed/signed off by the GB, where I read from @caniszczyk's comment above that only the first section needed to be/was approved.

**Pull request merged**
